### PR TITLE
D8NID-146

### PR DESCRIPTION
**Pull request by David-Corbett**
create a new attribute variable for content attributes and if there are any variables declared, wrap page.content with an additional div

### CODE REVIEW

File: nicsdru_origins_theme.theme [hunk 1]
```diff
@@ -79,6 +79,7 @@ function nicsdru_origins_theme_preprocess_page(array &$variables) {
     'class' => ['layout-sidebar-first'],
     'id' => ['first_sidebar']
   ];
+  $variables['content_content_attr'] = [];
```
Comment (johangant) [inline nicsdru_origins_theme.theme]: `content_content`? Is there a more descriptive reference we can have here? Main content/central content/wrapper... naming stuff is hard :)

**Review by johangant — COMMENTED** [review]
👍 just the one question about the naming of `content_content` :)

File: nicsdru_origins_theme.theme (hunk 1 above)
Comment (David-Corbett) [reply nicsdru_origins_theme.theme]: refactored to 'content_wrapper_attr'

File: templates/layout/page.html.twig [hunk 2]
```diff
@@ -65,7 +65,13 @@
       {{- page.sidebar_first -}}
     </div>
   {%- endif -%}
-  {{- page.content -}}
+  {% if content_content_attr is not empty %}
```
Comment (johangant) [inline templates/layout/page.html.twig]: should be `content_wrapper_attr` instead of `content_content_attr`

File: nicsdru_origins_theme.theme [hunk 3]
```diff
@@ -87,6 +88,7 @@ function nicsdru_origins_theme_preprocess_page(array &$variables) {
   // Declare region attributes.
   $variables['main_content_attr'] = new Attribute($variables['main_content_attr']);
   $variables['sidebar_first_attr'] = new Attribute($variables['sidebar_first_attr']);
+  $variables['content_content_attr'] = new Attribute($variables['content_content_attr']);
```
Comment (johangant) [inline nicsdru_origins_theme.theme]: s/content_content_attr/content_wrapper_attr :)

**Review by johangant — CHANGES_REQUESTED** [review]
Some missed variable renamings, otherwise 👍

**Review by johangant — APPROVED** [review]
Looks good now, thanks :shipit:

**Pull request merged**